src/main.cpp

```diff
@@ -432,13 +458,19 @@ int main(const int argc, char**const argv)
       // Input a tournament file, and compute the pairings of the next round.
       try
       {
-        std::ifstream inputStream(inputFilename);
-
         // Read the tournament.
         tournament::Tournament tournament;
         try
         {
-          tournament = fileformats::trf::readFile(inputStream, true);
+          if (doPairingsWithoutFilename)
+          {
+            tournament = fileformats::trf::readFile(std::cin, false);
```

Comment:
The boolean should be true

Reply:
You are right, thank you! Corrected via d00d2196297825c6c9e05fe7cdee5e7c6fb0b3ae.